### PR TITLE
Updating Dataflow API version to revision 235.

### PR DESCRIPTION
**Pull request by pabloem**
r: @tgroh

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -114,7 +114,7 @@
     <cloudresourcemanager.version>v1-rev6-1.22.0</cloudresourcemanager.version>
     <pubsubgrpc.version>0.1.18</pubsubgrpc.version>
     <clouddebugger.version>v2-rev8-1.22.0</clouddebugger.version>
-    <dataflow.version>v1b3-rev221-1.22.0</dataflow.version>
+    <dataflow.version>v1b3-rev235-1.22.0</dataflow.version>
```
Comment (lukecwik) [inline pom.xml]: Please update the build.gradle file as well.

Comment (pabloem) [reply pom.xml]: Done. Thanks for the catch.

**Comment by pabloem** [comment]
Jenkins retest this please

**Comment by pabloem** [comment]
@lukecwik PTAL : )

**Comment by pabloem** [comment]
Closing for now.

**Pull request closed without merging**